app/views/admin/invoices/show.html.erb

```diff
@@ -2,6 +2,25 @@
 <h1> Invoice #<%=@invoice.id%> </h1>
 
 <p> Status: <%= @invoice.status %> </p>
-<p> Invoice Date: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %> </p>
-<p> Customer: <%= @invoice.customer.first_name %> <%= @invoice.customer.last_name %> </p>
+<p> Created on: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %> </p>
+<p> Total Revenue: $<%= @invoice.total_invoice_revenue_dollars %>
```

Comment:
i think using a number_with_precision method would look better that way you can add a delimiter for better formating.

[number_with_precision](https://apidock.com/rails/ActionView/Helpers/NumberHelper/number_with_precision)

Reply:
It will also keep the decimal at ".00" so that we won't get numbers that are only one value after the decimal